main/2024/CG-12-17.md

```diff
@@ -0,0 +1,31 @@
+![WebAssembly logo](/images/WebAssembly.png)
+
+## Agenda for the December 17th video call of WebAssembly's Community Group
+
+- **Where**: Virtual meeting
+- **When**: 2024-12-17, 17:00-18:00 UTC (2024-12-17, 9am-10am PDT, 18:00-19:00 CEST)
```

Comment:
```suggestion
- **When**: 2024-12-17, 17:00-18:00 UTC (2024-12-17, 9am-10am PST, 18:00-19:00 CEST)
```

Reply:
I should have just written a script for that...

Reply:
If we had a script that took a group name, a start date, and a number of meetings to generate files for, I certainly wouldn't complain...